csfieldguide/chapters/urls.py

```diff
@@ -24,11 +24,16 @@
         views.glossary_json,
         name="glossary_json"
     ),
-    # eg: /chapters/algorithms/
+    # eg: /chapters/introduction/
```

Comment:
Would be great to change this example to URL and the one on line 33 to a chapter that isn't labelled 'introduction' as I initially thought this was referring to the introduction attribute of the model. Something like 'algorithms' would avoid that short confusion.